src/couch.js

```diff
@@ -12,23 +15,66 @@ async function _request(token, path, options) {
   return await response.json();
 }
 
+async function _requestAll(token, path, content) {
+  payload =
+    `{
+    "keys": [` +
+    content +
+    `]
+  }`;
```

Comment:
You'd be better off keeping track of the payload as an object, and then doing `JSON.stringify(payload)` when you insert it into the fetch body.